frappe/core/doctype/server_script/test_server_script.py

```diff
@@ -81,6 +81,7 @@ def setUpClass(cls):
 	def tearDownClass(cls):
 		frappe.db.commit()
 		frappe.db.sql('truncate `tabServer Script`')
+		frappe.cache().delete_key('server_script_map')
 
 	def setUp(self):
 		frappe.cache().delete_value('server_script_map')
```

Comment:
Any idea why this is in `setUp` in the first place?

Reply:
nope